automated/build.sh

```diff
@@ -5,6 +5,12 @@ set -eu +x -o pipefail
 ROOT=$(cd $(dirname $0) ; pwd)
 TARGET=$1
 
+# Home on the container
+: ${C_HOME:=$HOME}
```

Comment:
It has always annoyed me that we have a different path in the container compared to the host.

Though this change seems mostly cosmetic?

Reply:
Yes, this just allowed me to override the variable in testing, but doesn't do anything different on its own.